pkg/capacity/capacity.go

```diff
@@ -143,10 +273,21 @@ func getPodMetrics(mClientset *metrics.Clientset, namespace string) *v1beta1.Pod
 	return pmList
 }
 
-func getNodeMetrics(mClientset *metrics.Clientset, nodeLabels string) *v1beta1.NodeMetricsList {
+func getNodeMetrics(mClientset *metrics.Clientset, nodeList *corev1.NodeList, nodeLabels string, nodeTaints string) *v1beta1.NodeMetricsList {
 	nmList, err := mClientset.MetricsV1beta1().NodeMetricses().List(context.TODO(), metav1.ListOptions{
 		LabelSelector: nodeLabels,
 	})
+
+	if nodeTaints != "" {
+		taintedNodeList := *nodeList
+		taintedNmList := removeNodeMetricsWithTaints(nmList, taintedNodeList)
```

Comment:
Maybe a dumb question, but if we're just doing a join with the nodes later, and we're already removing the nodes that are tainted separately, do we need to actually remove the node metrics? I'd thought that might just automatically happen if corresponding nodes didn't exist, but it's been awhile since I've worked with this code.

Reply:
I believe you are correct. I tested `kube-capactiy -t` with and without this block of code and the results are the same.